tests/llms/test_openai_like.py

```diff
@@ -2,56 +2,98 @@
 from unittest.mock import patch
 
 from llama_index.llms import OpenAILike
-from llama_index.llms.base import ChatMessage
+from llama_index.llms.base import ChatMessage, MessageRole
+from llama_index.llms.openai import Tokenizer
+from llama_index.llms.openai_like import localai_defaults
 
 
-class MockTokenizer:
-    def encode(self, text: str) -> List[str]:
```

Comment:
This type hint wasn't matching the `Tokenizer` type, subclassing `Tokenizer` exposes this.

Also, just fyi when naming these things, it's not a `MagicMock` or `Mock` subclass, so they should be named "stub."  Read here for more info: https://martinfowler.com/articles/mocksArentStubs.html

Cc @Disiok

Reply:
Thanks for the fix!